egs/babel/s5d/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,336 @@
+#! /bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+# Features configs
+feat_type=plp
+feat_config=conf/plp.conf
+add_pitch=true
+pitch_config=conf/pitch.conf
+
+# Uniform segmentation options
+max_segment_duration=10
+overlap_duration=2.5
+max_remaining_duration=5
+
+# Decoding options
+lang=data/lang
+lang_test=data/lang
+
+data_dir=data/train
+sat_model_dir=/export/b17/jtrmal/babel/104-pashto-flp80-p-ext/exp/tri5_cleaned
+model_dir=/export/b17/jtrmal/babel/104-pashto-flp80-p-ext/exp/tri4
+graph_dir=
+
+affix=_1a
+stage=-1
+nj=80
+
+. path.sh
+. cmd.sh 
+
+set -e -u -o pipefail
+. utils/parse_options.sh 
+
+if [ $# -ne 0 ]; then
+  exit 1
+fi
+
+dir=exp/segmentation${affix}
+mkdir -p $dir
+
+cat <<EOF > $dir/garbage_words.txt
+<hes>
+<noise>
+<v-noise>
+EOF
+
+cat <<EOF > $dir/silence_words.txt
+<silence>
+EOF
+
+function make_mfcc {
```

Comment:
I don't really like having so many options in a script in local/.  Since it's already hardcoded in the script itself which type of features to use, I think it would be preferable to just have the script directly call the commands it needs to call, without too much capacity for configuration.  Once you figure out the best configuration, I think would be best to just hardcode that in this script.